test/gtest/ucp/test_ucp_am.cc

```diff
@@ -599,6 +620,26 @@ UCS_TEST_P(test_ucp_am_nbx, rx_rts_reply_am_on_closed_ep, "RNDV_THRESH=32K")
     test_recv_on_closed_ep(64 * UCS_KBYTE, UCP_AM_SEND_REPLY);
 }
 
+UCS_TEST_P(test_ucp_am_nbx, rx_persistent_data)
+{
+    void *data_desc = NULL;
+    char data       = 'd';
+
+    set_am_data_handler(receiver(), TEST_AM_NBX_ID, am_data_hold_cb, &data_desc,
+                        UCP_AM_FLAG_PERSISTENT_DATA);
+
+    ucp_request_param_t param;
+
+    param.op_attr_mask    = 0ul;
+    ucs_status_ptr_t sptr = ucp_am_send_nbx(sender().ep(), TEST_AM_NBX_ID, NULL,
+                                            0ul, &data, sizeof(data), &param);
+    wait_for_flag(&data_desc);
+    EXPECT_TRUE(data_desc != NULL);
```

Comment:
1. rename data_desc to smth else to avoid confusion with UCP internal descs
2. compare *data_desc to data

Reply:
renamed